src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -1073,6 +1081,25 @@ public enum HitTestMode
             All = Default | NodeData | ImageData,
         }
 
+        /// <summary>
+        /// WebEngine type which can be set by a specific constructor of this WebView.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum WebEngineType
+        {
+            /// <summary>
+            /// Chromium Web Engine type.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Chromium = 0,
```

Comment:
```suggestion
            UseSystemSetting = -1,
            Chromium = 0,
```

We'd better make one more enum s.t. allow to use system setting.

Since dali can choose the web engine type by environment value : `DALI_WEB_ENGINE_NAME`

For current implementation, we will use LWE if we export `DALI_WEB_ENGINE_NAME=lwe` even if we set Chromium type.

So please make this web engine type enum will choose web engine forcibly, and ignore environment internally.

Reply:
Thank you very much for your review!
I have commit your suggestion.
thank you.